.devcontainer/devcontainer.json

```diff
@@ -2,19 +2,15 @@
 // README at: https://github.com/devcontainers/templates/tree/main/src/rust
 {
 	"name": "qsharp",
-	"image": "mcr.microsoft.com/devcontainers/rust:dev-bullseye",
+	"image": "mcr.microsoft.com/devcontainers/python:3",
 	"features": {
 		"ghcr.io/devcontainers/features/node:1": {
 			"nodeGypDependencies": true,
 			"version": "lts"
 		},
-		"ghcr.io/devcontainers/features/python:1": {
-			"installTools": true,
-			"installJupyterlab": true,
-			"version": "3.11"
-		}
+		"ghcr.io/devcontainers/features/rust:1": {}
```

Comment:
Now that you start with a non-Rust base image, will this step not fetch the latest crates.io index as part of bootstrapping Rust? Do we still need `cargo update`?

Reply:
Sadly no, from what I observed rustup will install cargo and pull down a matching cached index of crates.io, so it does not come with the latest index by default. If I'm wrong and the behavior I observed was caused by something else, then the `cargo update --dry-run` will be mostly benign and just around for people in weird corner cases, but either way it's a nice to have with low cost.